src/admin/components/ResourceTabs/InformationTab/components/SiteInformationAside/QuickActions.tsx

```diff
@@ -41,7 +43,11 @@ const QuickActions: FC = () => {
         <Button
           className="button-aside-page-admin"
           component={Link}
-          to={createPath({ resource: modules.project.ResourceName, type: "show", id: record.project.uuid })}
+          to={createPath({
+            resource: modules.project.ResourceName,
+            type: "show",
+            id: record?.project?.uuid ?? record?.projectUuid
```

Comment:
Since this one is only used for sites, `record?.projectUuid` should be fine here.

Reply:
changed